omniscidb/CMakeLists.txt

```diff
@@ -579,6 +579,15 @@ endif()
 # RapidJSON
 include_directories(ThirdParty/rapidjson)
 add_definitions(-DRAPIDJSON_HAS_STDSTRING)
+if(NOT MSVC)
```

Comment:
I believe the fix is required for non-Windows builds -- the vcpkg version on Windows is too old for the fix to apply there as of this writing (which may have been some time ago). But, msvc doesn't seem to have an issue with the iterator format used in rapidjson -- perhaps because they are not enforcing all parts of c++17 yet?

Reply:
Do you want to apply the same fix in the top-level `CMakeLists.txt`?